multimodal/src/autogluon/multimodal/utils.py

```diff
@@ -1976,3 +1977,37 @@ def update_config_by_rules(
                 UserWarning,
             )
     return config
+
+
+def process_save_path(path, resume: Optional[bool] = False, raise_if_exist: Optional[bool] = True):
+    """
+    Convert the provided path to an absolute path and check whether it is valid.
+    If a path exists, either raise error or return None.
+    A None path can be identified by the `setup_outputdir` to generate a random path.
```

Comment:
What does the random path look like?

Reply:
Something like f"AutogluonModels/ag-{timestamp}"